<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCodeClean</class>
 <widget class="QWidget" name="frmCodeClean">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="txtFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelect">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDo">
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtCode"/>
   </item>
   <item>
    <widget class="QTextEdit" name="txtResult"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
